### PR TITLE
[ZT] update load balancer page

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

File: content/cloudflare-one/connections/connect-apps/routing-to-tunnel/lb.md [hunk 1]
```diff
@@ -1,65 +1,61 @@
...
-```sh
-$ cloudflared tunnel route lb <tunnel ID or NAME> <load balancer name> <load balancer pool>
+```bash
+$ cloudflared tunnel route lb <tunnel name/uuid> <hostname> <load balancer pool>
 ```
 
-**Note**: this command requires the `cert.pem` file.
+* `<hostname>`: the DNS hostname of the load balancer, for example `lb.example.com`.
+
+* `<load balancer pool>`: the ID of the [pool](/load-balancing/how-to/create-pool/#create-a-pool) that will contain the tunnel subdomain.
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-apps/routing-to-tunnel/lb.md:33]: check if it's ID or name

File: content/cloudflare-one/connections/connect-apps/routing-to-tunnel/lb.md [hunk 2]
```diff
@@ -1,65 +1,61 @@
...
 
-A load balancer maintains [session affinity](/load-balancing/understand-basics/session-affinity/) by treating an entire Named Tunnel as an origin server, meaning that it does not distinguish between Named Tunnels [running as replicas](/cloudflare-one/connections/connect-apps/run-tunnel/deploy-cloudflared-replicas/).
+### Session affinity and replicas
 
-To maintain session affinity for individual service instances running behind tunnel replicas, use different Named Tunnel IDs.
+The load balancer does not distinguish between [replicas](/cloudflare-one/connections/connect-apps/run-tunnel/deploy-cloudflared-replicas/) of the same tunnel. If you run the same tunnel UUID on two separate hosts, the load balancer treats both hosts as a single origin server. To maintain [session affinity](/load-balancing/understand-basics/session-affinity/) between a client and a particular host, you will need to connect each host to Cloudflare using a different tunnel UUID.
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-apps/routing-to-tunnel/lb.md]: does this make sense?

File: content/cloudflare-one/connections/connect-apps/routing-to-tunnel/lb.md [hunk 3]
```diff
@@ -1,65 +1,61 @@
...
 
 ### Local connection preference
 
 If you notice traffic imbalances across origin servers in different locations, you may have to adjust your load balancer setup.
 
-`cloudflared` connections give preference to tunnels that terminate in the same data center (local connections). This behavior can impact how connections are weighted and traffic is distributed.
+`cloudflared` connections give preference to tunnels that terminate in the same Cloudflare data center. This behavior can impact how connections are weighted and traffic is distributed.
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-apps/routing-to-tunnel/lb.md:65]: Really confused by this section. What does it mean to "give preference" to a tunnel? Doesn't `cloudflared` only create one tunnel, not multiple tunnels? How do we know where a given tunnel will terminate?

File: content/cloudflare-one/connections/connect-apps/routing-to-tunnel/lb.md [hunk 4]
```diff
@@ -1,65 +1,61 @@
...
 
 The solution depends on the type of tunnel being used. If running Classic Tunnels, put your origins in different pools. If running [Named Tunnels replicas](/cloudflare-one/connections/connect-apps/run-tunnel/deploy-cloudflared-replicas/) (using a shared ID), switch to separate Named Tunnels as distinct origins.
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-apps/routing-to-tunnel/lb.md]: What is a Classic Tunnel??

File: content/cloudflare-one/connections/connect-apps/routing-to-tunnel/lb.md [hunk 5]
```diff
@@ -1,65 +1,61 @@
...
 
-A load balancer maintains [session affinity](/load-balancing/understand-basics/session-affinity/) by treating an entire Named Tunnel as an origin server, meaning that it does not distinguish between Named Tunnels [running as replicas](/cloudflare-one/connections/connect-apps/run-tunnel/deploy-cloudflared-replicas/).
+### Session affinity and replicas
 
-To maintain session affinity for individual service instances running behind tunnel replicas, use different Named Tunnel IDs.
+The load balancer does not distinguish between [replicas](/cloudflare-one/connections/connect-apps/run-tunnel/deploy-cloudflared-replicas/) of the same tunnel. If you run the same tunnel UUID on two separate hosts, the load balancer treats both hosts as a single origin server. To maintain [session affinity](/load-balancing/understand-basics/session-affinity/) between a client and a particular host, you will need to connect each host to Cloudflare using a different tunnel UUID.
 
 ### Local connection preference
 
 If you notice traffic imbalances across origin servers in different locations, you may have to adjust your load balancer setup.
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-apps/routing-to-tunnel/lb.md:63]: How do you "adjust your load balancer setup"? A concrete example would be helpful.

**Comment by ranbel** [comment]
@abelinkinbio

File: content/cloudflare-one/connections/connect-apps/install-and-setup/deploy-cloudflared-replicas.md [hunk 6]
```diff
@@ -9,32 +9,19 @@ weight: 2
...
+## `cloudflared` replicas
 
 Cloudflare Tunnel also allows users to deploy additional instances of our connector, `cloudflared`, for availability and failover scenarios. We refer to these unique instances as replicas. Each replica establishes four new connections which serve as additional points of ingress to your origin, should you need them. Each of the replicas will point to the same tunnel. This ensures that your network remains up in the event a single host running `cloudflared` goes down.
 
 By design, replicas do not offer any level of traffic steering (random, hash, or round-robin). Instead, when a request arrives to Cloudflare, the network will pick any connection available to the origin. If a connection fails, we will retry others, but there is no guarantee about which connection is chosen.
 
-## When to use `cloudflared` replicas
+### When to use `cloudflared` replicas
 
 - To provide additional points of availability for a single tunnel
```
Comment (angelampcosta) [inline content/cloudflare-one/connections/connect-apps/install-and-setup/deploy-cloudflared-replicas.md]:
```suggestion
- To provide additional points of availability for a single tunnel.
```

File: content/cloudflare-one/connections/connect-apps/install-and-setup/deploy-cloudflared-replicas.md [hunk 7]
```diff
@@ -9,32 +9,19 @@ weight: 2
...
 - To allocate failover nodes within your network
```
Comment (angelampcosta) [inline content/cloudflare-one/connections/connect-apps/install-and-setup/deploy-cloudflared-replicas.md]:
```suggestion
- To allocate failover nodes within your network.
```

File: content/cloudflare-one/connections/connect-apps/install-and-setup/deploy-cloudflared-replicas.md [hunk 8]
```diff
@@ -9,32 +9,19 @@ weight: 2
...
 - To update the configuration of a tunnel without downtime
```
Comment (angelampcosta) [inline content/cloudflare-one/connections/connect-apps/install-and-setup/deploy-cloudflared-replicas.md]:
```suggestion
- To update the configuration of a tunnel without downtime.
```

File: content/cloudflare-one/connections/connect-apps/install-and-setup/deploy-cloudflared-replicas.md [hunk 9]
```diff
@@ -72,18 +59,19 @@ For information about running `cloudflared` instances in a Kubernetes deployment
...
 
-This command creates an LB DNS record that points the specified hostname to the subdomain of your tunnel (`UUID.cfargotunnel.com`). Traffic will not be proxied unless the tunnel is running.
+### When to use load balancers
 
-{{<Aside type="note">}}
-In order to create DNS records using `cloudflared`, the [`cert.pem`](/cloudflare-one/connections/connect-apps/install-and-setup/tunnel-useful-terms/#certpem) file must be installed on your system.
-{{</Aside>}}
-  
-For more information, refer to our [Cloudflare Load Balancing](/load-balancing/) documentation.
+- To intelligently steer traffic based on latency, geolocation, or other signals
```
Comment (angelampcosta) [inline content/cloudflare-one/connections/connect-apps/install-and-setup/deploy-cloudflared-replicas.md]:
```suggestion
- To intelligently steer traffic based on latency, geolocation, or other signals.
```

File: content/cloudflare-one/connections/connect-apps/install-and-setup/deploy-cloudflared-replicas.md [hunk 10]
```diff
@@ -72,18 +59,19 @@ For information about running `cloudflared` instances in a Kubernetes deployment
...
+- To implement failover logic if a tunnel reaches an inactive state
```
Comment (angelampcosta) [inline content/cloudflare-one/connections/connect-apps/install-and-setup/deploy-cloudflared-replicas.md]:
```suggestion
- To implement failover logic if a tunnel reaches an inactive state.
```

File: content/cloudflare-one/connections/connect-apps/install-and-setup/deploy-cloudflared-replicas.md [hunk 11]
```diff
@@ -72,18 +59,19 @@ For information about running `cloudflared` instances in a Kubernetes deployment
...
+- To get alerted when a tunnel reaches an inactive state
```
Comment (angelampcosta) [inline content/cloudflare-one/connections/connect-apps/install-and-setup/deploy-cloudflared-replicas.md]:
```suggestion
- To get alerted when a tunnel reaches an inactive state.
```

File: content/cloudflare-one/connections/connect-apps/install-and-setup/deploy-cloudflared-replicas.md [hunk 12]
```diff
@@ -72,18 +59,19 @@ For information about running `cloudflared` instances in a Kubernetes deployment
...
+- To distribute traffic more evenly across your Cloudflare Tunnel-accessible origins
```
Comment (angelampcosta) [inline content/cloudflare-one/connections/connect-apps/install-and-setup/deploy-cloudflared-replicas.md]:
```suggestion
- To distribute traffic more evenly across your Cloudflare Tunnel-accessible origins.
```

**Review by angelampcosta — APPROVED** [review]

**Pull request merged**